### PR TITLE
adding support.md

### PR DESCRIPTION
**Pull request by uday31in**
**This PR fixes**

adding support.md

### CODE REVIEW

**Review by krnese — APPROVED** [review]

**Review by kwill-MSFT — APPROVED** [review]

**Review by krnese — APPROVED** [review]

**Comment by uday31in** [comment]
@alex-frankel - can you please review?

File: support.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+- ARM Deployment Issues e.g. template validation, CheckAccess API etc.
+- Git Actions/Azure DevOps pipeline itself however please note that PowerShell module will be supported via community support only.
+
+Any issues that deemed outside of the above list by Microsoft support and/or requires bugfix in the Template or Code in the repo, Microsoft support will redirect user to file the issue on GitHub.
+
+Project maintainers and community aim to get issues resolved in timely fashion as per community support policy of this repo.
+
+## Community Support Policy
+
+Project maintainers will aim to respond within 3 business days to get a meaningful response for any new issues.
```
Comment (alex-frankel) [inline support.md:20]: I think the only challenge here is if I am a large enterprise looking to take a dependency on AzOps, this support policy does not inspire a lot of confidence that my issue will get resolved quickly.

File: support.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+Following list of issues are within the scope of Microsoft support:
+
+- Tenant, Management Group, Subscription and Resource Group deployment
+- Az PowerShell commandlets e.g. Get-AzManagementGroup
+- ARM Deployment Issues e.g. template validation, CheckAccess API etc.
+- Git Actions/Azure DevOps pipeline itself however please note that PowerShell module will be supported via community support only.
+
+Any issues that deemed outside of the above list by Microsoft support and/or requires bugfix in the Template or Code in the repo, Microsoft support will redirect user to file the issue on GitHub.
+
+Project maintainers and community aim to get issues resolved in timely fashion as per community support policy of this repo.
```
Comment (alex-frankel) [inline support.md:16]: This sentence feels like it should be under the "Community Support Policy"

**Review by alex-frankel — APPROVED** [review]
Non-blocking feedback

**Pull request merged**